lambdas/pkgselect/index.py

```diff
@@ -172,7 +178,8 @@ def lambda_handler(request):
         # Call s3 select to fetch only logical keys matching the
         # desired prefix (folder path)
         prefix_length = len(prefix) if prefix is not None else 0
-        sql_stmt = f"SELECT SUBSTRING(s.logical_key, {prefix_length + 1}) AS logical_key FROM s3object s"
+        sql_stmt = f"SELECT SUBSTRING(s.logical_key, {prefix_length + 1}) AS logical_key"
+        sql_stmt += ", s.\"size\", s.physical_keys[0] as physical_key FROM s3object s"
```

Comment:
```suggestion
        sql_stmt = (
            f"SELECT SUBSTRING(s.logical_key, {prefix_length + 1}) AS logical_key"
            ", s.\"size\", s.physical_keys[0] as physical_key FROM s3object s"
        )
```

Reply:
We might as well send back the physical key as a list for consistency with manifest format? i.e. then JS can reuse any manifest-processing code.

Reply:
@akarve i think it's actually cleaner this way (with single physical_key)